platform/quality-assurance/deliverables.md

```diff
@@ -0,0 +1,21 @@
+# Deliverables
+A list of deliverables and explanation of their contents is listed below.
+
+## Quality Assurance Deliverables
+1. Test Plan
+    - This artifact usually takes the form of a Word / Google Doc. The test plan artifact is an explanation of the functional changes being developed and a plan for how to test each change.
+    - Example: [Test Plan Template](https://docs.google.com/document/d/10dYOWyHSTGB_gKiPZSdcdq6fwjrlRhBsK3yUIyS3Gc0/edit?usp=sharing)
+2. Test Cases
+    - An export from your test case management software of choice that contains each test case used in your test plan.
+    - In the near future, VSP hopes to offer a test case management software that all VFS teams will have access to.
+    - Example: [Test Case Template](https://drive.google.com/file/d/1yQpyUnt9yoNt_dRCRWW2DIMfRYglCRwT/view?usp=sharing)
+3. Test Case Execution Logs
+    - An export from your test case management software of choice that contains the pass/fail runs for each of the test cases used in your test plan.
+    In the near future, VSP hopes to offer a test case management software that all VFS teams will have access to.
+    - Example: [Test Execution Example](https://drive.google.com/file/d/1_DeDS8lE2yxjcYPd_TEiSANl65Fsn95s/view?usp=sharing)
+4. Requirements Traceability Matrix (RTM)
+    - This artifact usually takes the form of a Word / Google Doc. The RTM artifact is a mapping from requirement / user story to test case that demonstrates how the functional changes in your product were tested for this release.
+    - Example: [RTM Template](https://drive.google.com/file/d/1fAWBuscDAGLLC18C5lbekjvjGElAaAm4/view?usp=sharing)

```

Comment:
can you confirm this file (and any other Drive files linked here) are in the VSP Team drive?

Reply:
They are now!